src/core_plugins/tile_map/public/kibana_map.js

```diff
@@ -398,7 +398,7 @@ export class KibanaMap extends EventEmitter {
     const southEast = bounds.getSouthEast();
     const northWest = bounds.getNorthWest();
     if (
-      southEast.lng === northWest.lng &&
+      southEast.lng === northWest.lng ||
```

Comment:
why this change ?

Reply:
the and-case didn't trap the case where the visualization would have a width, but not a height (or vice-versa).